src/test/resources/forms/release_notify_code_freeze.form

```diff
@@ -26,5 +26,7 @@
     }
   ],
   "type": "default",
-  "id": "notify_code_freeze_from"
+  "id": "notify_code_freeze_from",
+  "executionPlatform": "Camunda Cloud",
+  "executionPlatformVersion": "1.2"
```

Comment:
❓ What is this used for? You did notice update the JSON in the user task so as of now this doesn't do anything.